server/ai_mediaserver.go

```diff
@@ -360,3 +361,26 @@ func (ls *LivepeerServer) ImageToVideoResult() http.Handler {
 		_ = json.NewEncoder(w).Encode(resp)
 	})
 }
+
+func (ls *LivepeerServer) StartLiveVideo() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		streamName := r.FormValue("stream")
+		if streamName == "" {
+			http.Error(w, "Missing stream name", http.StatusBadRequest)
+			return
+		}
+		requestID := string(core.RandomManifestID())
+		params := aiRequestParams{
+			node:        ls.LivepeerNode,
+			os:          drivers.NodeStorage.NewSession(requestID),
+			sessManager: ls.AISessionManager,
+		}
+		ctx := clog.AddVal(r.Context(), "request_id", requestID)
+		clog.Infof(ctx, "Received live video AI request")
+		req := worker.StartLiveVideoToVideoFormdataRequestBody{
+			// TODO set model and initial parameters here if necessary (eg, prompt)
+		}
+		resp, err := processAIRequest(ctx, params, req)
+		fmt.Println("stream", streamName, "resp", resp, "error", err)
```

Comment:
If we want to merge it into `ai-video`, then please remove or change to `glog` / `clog`.

Reply:
changed to clog in https://github.com/livepeer/go-livepeer/commit/9e461a7d2ee59b1cd5603382d403c082b71469ad

btw I might send a PR in the near future to add slog-style parameter lists to clog since they are super convenient for ad-hoc KV pairs (in addition to the pre-defined `clog.Add` method we have)